internal/rpc/user_device_data_api.go

```diff
@@ -85,13 +86,54 @@ func (s *userDeviceData) GetUserDeviceData(ctx context.Context, req *pb.UserDevi
 	}, nil
 }
 
+// ? TODO: should this follow mediator patter as all other services?
```

Comment:
no - actually trying to get away from mediatr in all repos except for device definitions, where it is entrenched

Reply:
thanks, removing this TODO then.